lib/osc-loader.js

```diff
@@ -27,6 +28,10 @@ export default class OscLoader extends EventEmitter {
             let resultMap = osc.fromBuffer(msg);
 
             if (resultMap.address === this.address) {
+                if (resultMap.args.length > 1) {
+                  Editor.goTo(resultMap.args[1].value)
```

Comment:
I guess you should decrement the row value by one.
When I evaluate this in SuperCollider:
```
l = NetAddr("127.0.0.1", 3333);

l.sendMsg("/atom/eval", "line", 68)
```

Then I trigger the atom line `69`. I guess this happens, because the atom lines in the ui starts by 1 and not by 0.

Reply:
Yes, I specified in the readme, don't know what behaviour would be the best. 
I'm used to count stuff starting from zero, but actually atom starts by 1.
Maybe is better to give to the use an 1:1 relationship between atom row number and the message row number